app/controllers/oauth_controller.rb

```diff
@@ -53,17 +46,14 @@ def callback
       # Catch the error and proceed to redirenct to '/' if we fail to get the access token
       begin
         # Get the access token and initialize it with ACCESS_TOKEN class
-        self.class.token = OAUTH_CLIENT.auth_code.get_token(params[:code], :redirect_uri => ENV['OAUTH_CALLBACK_URL'])
+        token = OAUTH_CLIENT.auth_code.get_token(params[:code], :redirect_uri => ENV['OAUTH_CALLBACK_URL'])
```

Comment:
If we're really removing all calls to self.class.token, we can probably get rid of the above attr_accessor :token bit, since it's not really needed anymore.

Reply:
I totally agree! I have removed it!